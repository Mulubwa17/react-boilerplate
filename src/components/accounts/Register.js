import React from "react";
import { Link } from "react-router-dom";

function Register() {
  return (
    <p className="example-div">
      Welcome to the Register Page{" "}
      <Link to="/login" style={{ textDecoration: "none", color: "#79ad6f" }}>
        {" "}
        Login Here
      </Link>
    </p>
  );
}

export default Register;
