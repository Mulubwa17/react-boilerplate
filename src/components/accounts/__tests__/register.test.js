import React from "react";
import { shallow } from "enzyme";
import Register from "../Register";

it("should render without crashing", () => {
  shallow(<Register />);
});
it("should render correctly ", () => {
  const registerSnap = shallow(<Register />);
  expect(registerSnap).toMatchSnapshot();
});
