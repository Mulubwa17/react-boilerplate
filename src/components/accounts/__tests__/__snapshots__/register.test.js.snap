// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly  1`] = `
<p
  className="example-div"
>
  Welcome to the Register Page
   
  <Link
    replace={false}
    style={
      Object {
        "color": "#79ad6f",
        "textDecoration": "none",
      }
    }
    to="/login"
  >
     
    Login Here
  </Link>
</p>
`;
